src/seedu/addressbook/commands/IncorrectCommand.java

```diff
@@ -0,0 +1,20 @@
+package seedu.addressbook.commands;
+
+
+/**
+ * Represents an incorrect command. Upon execution, produces some feedback to the user.
+ */
+public class IncorrectCommand extends Command{
+
+    String feedbackToUser;
+
+    public IncorrectCommand(String feedbackToUser){
+        this.feedbackToUser = feedbackToUser;
+    }
+
+    @Override
+    public CommandResult execute() {
+        return new CommandResult(feedbackToUser);
+    }
```

Comment:
It's better to user a normal object rather than an exception to handle user mistakes. They are not truly exceptional situations.